src/test/java/org/jenkins_ci/plugins/run_condition/core/CauseConditionTest.java

```diff
@@ -241,7 +263,8 @@ private void runtest(List<Cause> causes, RunCondition condition, boolean expecte
     private UserCause createUserCause(String userid) {
         Authentication a = new UsernamePasswordAuthenticationToken(userid, userid);
 
-        a = hudson.getSecurityRealm().getSecurityComponents().manager.authenticate(a);
+        a = j.jenkins.getSecurityRealm().getSecurityComponents().manager.authenticate(a);
```

Comment:
Crazy BTW. Use `User.get('userid').impersonate()` instead. (After ensuring there is a `dummySecurityManager()` at least.)